.github/workflows/ci.yml

```diff
@@ -43,6 +43,21 @@ jobs:
       - name: Test
         run: bundle exec rake smithy-client:spec
 
+  smithy-test-endpoints:
```

Comment:
It would be preferable to run these tests as part of smithy spec instead of making an entirely new task and CI.

Reply:
Discussed offline - I'd prefer they are separate and its simpler to keep them separate for now.  We can update later.

Reply:
Pushed a simplification, feel free to take or not.

Reply:
Thanks - simplification looks good.